tests/scripts/depends.py

```diff
@@ -490,6 +491,9 @@ def __init__(self, options, conf):
         build_command = [options.make_command, 'CFLAGS=-Werror -O2']
         build_and_test = [build_command, [options.make_command, 'test']]
         self.all_config_symbols = set(conf.settings.keys())
+        algs = {crypto_knowledge.Algorithm(symbol.replace('_WANT', '')): symbol
```

Comment:
Two comments. 

- If we need to strip `_WANT` for algs, and it is a legit use case rahter than a debugging one, it could be something that is done by the Algorithm class.
- What is happening with algs as a variable? We are setting it but I cannot see it being used in the lifecyle of `__init__`. Was it meant ot be a class variable?

Reply:
>What is happening with algs as a variable? We are setting it but I cannot see it being used in the lifecyle of `__init__`. Was it meant ot be a class variable?

It is used in a following commit. Several PRs needs this thus broke these changes apart to avoid to depend on each other.